<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Efficiency MC - Transformateur de flux</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab_mapping">
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Mapping</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Model source</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btn_parcourir_source">
                  <property name="text">
                   <string>Parcourir</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QTreeView" name="sourceView">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>16</horstretch>
                  <verstretch>16</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>1697</width>
                  <height>2000</height>
                 </size>
                </property>
                <property name="acceptDrops">
                 <bool>true</bool>
                </property>
                <property name="verticalScrollBarPolicy">
                 <enum>Qt::ScrollBarAlwaysOn</enum>
                </property>
                <property name="horizontalScrollBarPolicy">
                 <enum>Qt::ScrollBarAlwaysOn</enum>
                </property>
                <property name="sizeAdjustPolicy">
                 <enum>QAbstractScrollArea::AdjustIgnored</enum>
                </property>
                <property name="dragEnabled">
                 <bool>true</bool>
                </property>
                <property name="dragDropMode">
                 <enum>QAbstractItemView::DragOnly</enum>
                </property>
                <property name="alternatingRowColors">
                 <bool>true</bool>
                </property>
                <property name="selectionBehavior">
                 <enum>QAbstractItemView::SelectItems</enum>
                </property>
                <property name="horizontalScrollMode">
                 <enum>QAbstractItemView::ScrollPerPixel</enum>
                </property>
                <property name="animated">
                 <bool>false</bool>
                </property>
                <property name="allColumnsShowFocus">
                 <bool>true</bool>
                </property>
                <attribute name="headerCascadingSectionResizes">
                 <bool>true</bool>
                </attribute>
                <attribute name="headerStretchLastSection">
                 <bool>true</bool>
                </attribute>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Model cible</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btn_parcourir_target">
                  <property name="text">
                   <string>Parcourir</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QTreeView" name="targetView">
                <property name="verticalScrollBarPolicy">
                 <enum>Qt::ScrollBarAlwaysOn</enum>
                </property>
                <property name="horizontalScrollBarPolicy">
                 <enum>Qt::ScrollBarAlwaysOn</enum>
                </property>
                <property name="sizeAdjustPolicy">
                 <enum>QAbstractScrollArea::AdjustIgnored</enum>
                </property>
                <property name="alternatingRowColors">
                 <bool>true</bool>
                </property>
                <property name="selectionBehavior">
                 <enum>QAbstractItemView::SelectItems</enum>
                </property>
                <property name="allColumnsShowFocus">
                 <bool>true</bool>
                </property>
                <attribute name="headerCascadingSectionResizes">
                 <bool>false</bool>
                </attribute>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinAndMaxSize</enum>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPushButton" name="btn_save_mapping">
                <property name="text">
                 <string>Save</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_remove_mapping">
                <property name="text">
                 <string>Remove</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_add_mapping">
                <property name="text">
                 <string>Add</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_upload_mapping">
                <property name="text">
                 <string>Upload</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTableView" name="tableView">
              <property name="sizeAdjustPolicy">
               <enum>QAbstractScrollArea::AdjustToContents</enum>
              </property>
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <property name="showGrid">
               <bool>true</bool>
              </property>
              <property name="sortingEnabled">
               <bool>false</bool>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderStretchLastSection">
               <bool>false</bool>
              </attribute>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
       <zorder></zorder>
      </widget>
      <widget class="QWidget" name="tab_transformation">
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Transformation</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Input</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListView" name="list_input"/>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QToolButton" name="btn_remove_input">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="icon">
                 <iconset resource="icons.qrc">
                  <normaloff>:/inc/icons/inc/icons/delete.png</normaloff>:/inc/icons/inc/icons/delete.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="btn_add_input">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>...</string>
                </property>
                <property name="icon">
                 <iconset>
                  <selectedon>:/inc/icons/inc/icons/add.png</selectedon>
                 </iconset>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Mapping</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit_mapping"/>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="btn_upload_mapping_2">
              <property name="text">
               <string>Upload</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="btn_transform">
          <property name="text">
           <string>Transform</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>936</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="actionsMenu">
    <property name="title">
     <string>&amp;Actions</string>
    </property>
    <addaction name="insertRowAction"/>
    <addaction name="separator"/>
    <addaction name="removeRowAction"/>
    <addaction name="separator"/>
    <addaction name="insertChildAction"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="actionsMenu"/>
  </widget>
  <action name="exitAction">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="insertRowAction">
   <property name="text">
    <string>Insert Row</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I, R</string>
   </property>
  </action>
  <action name="removeRowAction">
   <property name="text">
    <string>Remove Row</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R, R</string>
   </property>
  </action>
  <action name="insertChildAction">
   <property name="text">
    <string>Insert Child</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>btn_parcourir_target</tabstop>
  <tabstop>lineEdit_mapping</tabstop>
  <tabstop>btn_upload_mapping_2</tabstop>
  <tabstop>btn_transform</tabstop>
  <tabstop>btn_parcourir_source</tabstop>
  <tabstop>targetView</tabstop>
  <tabstop>btn_upload_mapping</tabstop>
  <tabstop>sourceView</tabstop>
  <tabstop>btn_save_mapping</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
